src/PKSim.Core/Snapshots/Mappers/FormulationMapper.cs

```diff
@@ -0,0 +1,22 @@
+﻿using PKSim.Core.Model;
+using SnapshotFormulation = PKSim.Core.Snapshots.Formulation;
+using ModelFormulation = PKSim.Core.Model.Formulation;
+
+namespace PKSim.Core.Snapshots.Mappers
+{
+   public class FormulationMapper : ParameterContainerSnapshotMapperBase<ModelFormulation, SnapshotFormulation>
+   {
+      public FormulationMapper(ParameterMapper parameterMapper) : base(parameterMapper)
+      {
+      }
+
+      public override SnapshotFormulation MapToSnapshot(ModelFormulation modelFormulation)
```

Comment:
I think this should not be more complicated than that for basic container objects such as Events, Formulation, SimpleProtocol etc...